index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap" rel="stylesheet">
+    <title>Joseph Kwesi Kwentsir Portfolio</title>
+</head>
+<body>
+    <header class="page-header">
+      <nav class="nav-header"> 
+          <ul>
+              <li>Welcome</li>
```

Comment:
- [ ] Please wrap the home logo in an `s` tag because it will be used as a link to the home page

Reply:
Kindly confirm the tag to be used on the home logo. Do you mean the Welcome?